src/TodoListItem.jsx

```diff
@@ -0,0 +1,9 @@
+const TodoListItem = (props) => {
+  return (
+    <li key={props.todo.key}>
```

Comment:
Since key is included as a prop for the TodoListItem component, this list item element's key attribute can be removed.

Reply:
OK